telegram/ext/_utils/webhookhandler.py

```diff
@@ -50,21 +54,37 @@ class WebhookServer:
         "is_running",
         "_server_lock",
         "_shutdown_lock",
+        "unix",
     )
 
     def __init__(
-        self, listen: str, port: int, webhook_app: "WebhookAppClass", ssl_ctx: Optional[SSLContext]
+        self,
+        listen: DVType[str],
+        port: int,
+        webhook_app: "WebhookAppClass",
+        ssl_ctx: Optional[SSLContext],
+        unix: Optional[Union[str, Path]] = None,
     ):
+        if not isinstance(listen, DefaultValue) and unix:
+            raise RuntimeError(
+                "You can not pass unix and listen, only use one. Unix if you want to initialize a"
+                " unix socket, or listen for a standard TCP server"
+            )
```

Comment:
I would prefer for this to happen in `Updater` already. IMO it's not webhookhandlers responsibility do to argument parsing.

Another question/observation: IISC `port` and `url_path` are also mutually exclusive to `unix`, right? if so, these should be checked here as well and for this should also be `DefaultValue` objects I guess 🤔

Reply:
so much default